priv/repo/seeds.exs

```diff
@@ -9,3 +9,6 @@
 #
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
+EChronicler.Repo.insert!(%EChronicler.EChronicler.Journal_Entry{title: "Awesome Blog Post", author: "Grace", entry: "Test."})
+EChronicler.Repo.insert!(%EChronicler.EChronicler.Journal_Entry{title: "Cool Blog Post", author: "Grace", entry: "Test."})
+EChronicler.Repo.insert!(%EChronicler.EChronicler.Journal_Entry{title: "So-So Blog Post", author: "Grace", entry: "Test."})
```

Comment:
Not necessarily for this MR but just a note for your next story: it's useful for seed data to represent different variations on the data that you would expect. E.g. having an entry with very short text, an entry with a lot of text, etc.

Reply:
Added some longer entries!